packages/core/scripts/register-contract-umip/1_Propose.js

```diff
@@ -0,0 +1,92 @@
+// This script generates and submits a transaction that registers a new price requester for the DVM. It can be
+// run on a local ganache fork of the main net or can be run directly on the main net to execute the upgrade
+// transactions. To run this on the localhost first fork main net into Ganache with the proposerWallet unlocked as follows from core:
+// ganache-cli --fork https://mainnet.infura.io/v3/5f56f0a4c8844c96a430fbd3d7993e39 --unlock 0x2bAaA41d155ad8a4126184950B31F50A1513cE25 --unlock 0x7a3a1c2de64f20eb5e916f40d11b01c441b2a8dc --port 9545
+// Then execute the script from core:
+// yarn truffle exec ./scripts/register-contract-umip/1_Propose.js --network mainnet-fork --contract 0xCONTRACT_ADDRESS
```

Comment:
What address should we use for testing `0xCONTRACT_ADDRESS` for the Polygon tunnel?

Reply:
It would be the `OracleRootTunnel` address in the `networks/1.json` file. Do you think I should reference this in the comments?

Reply:
nope, was just checking for my own testing